load.Rmd

```diff
@@ -378,7 +382,6 @@ of 5.0%.
 ## Scenarios
 
 ```{r, include=FALSE}
-l$slc <- mod_logsqr$fitted.values
 l$base_olc <- l$Bicycle
 l$base_slc <- l$slc * l$All
 l$base_sic <- l$base_slc - l$base_olc
```

Comment:
Not connected with this PR but these names are pretty bad.  `base_slc` should be something like `gov_slc` and `base_olc` is surely just `olc`?